Source/JavaScriptCore/runtime/PropertyTable.cpp

```diff
@@ -65,7 +65,7 @@ PropertyTable::PropertyTable(VM& vm, unsigned initialCapacity)
     , m_keyCount(0)
     , m_deletedCount(0)
 {
-    ASSERT(isPowerOf2(m_indexSize));
+    ASSERT(std::has_single_bit(m_indexSize));
```

Comment:
Hm... this makes code less readable. Maybe we can keep `isPowerOf2` aliasing and just switch the underlying implementation?